USE-CASES/smartcity-geolocation.md

```diff
@@ -0,0 +1,127 @@
+## Title: Smart City Geolocation
+
+### Submitter(s): 
+
+Jennifer Lin (GovTech Singapore)
+
+### Reviewer(s):
+
+Michael McCool, Michael Lagally
+
+### Tracker Issue ID:
+
+<please leave blank>
+
+### Category:
+
+<please leave blank>
+
+### Class: 
+
+<please leave blank>
+
+### Status: 
+
+<please leave blank>
+
+### Target Users
+
+A Smart City managing mobile devices and sensors,
+including passively mobile sensor packs, packages,
+vehicles, and autonomous robots, where their location needs to
+be determined dynamically.
```

Comment:
Stakeholders/Users need to be further clarified. They include "city government, people counting service providers, police, network providers, ...